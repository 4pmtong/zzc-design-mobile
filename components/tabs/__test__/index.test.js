import React from 'react';
import {render} from 'enzyme';
import {renderToJson} from 'enzyme-to-json';
import Popup from '../index.tsx';
test( 'Modal', () => {
    const wrapper = render(
        <Popup
            maskClose={true}
            visible={true}
        >
            <h1>123</h1>
        </Popup>
    );
    expect( renderToJson( wrapper ) ).toMatchSnapshot();
} );

