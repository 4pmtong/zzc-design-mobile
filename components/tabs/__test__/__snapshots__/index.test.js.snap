// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal 1`] = `
<div
  class="zzc-tabs-box zzc-tabs-box-top"
>
  <div
    class="zzc-tabs-bar-box"
  >
    <div
      class="zzc-tabs-bar-wrap zzc-tabs-bar-wrap-am"
      style="touch-action:pan-x pan-y;transform:translate3d(0, 0px, 1px);-webkit-transform:translate3d(0, 0px, 1px);-moz-transform:translate3d(0, 0px, 1px)"
    >
      <div
        class="zzc-tabs-ls zzc-tabs-ls-horizontal"
      >
        <div
          class="zzc-tabs-ls-item zzc-tabs-active"
          style="width:33.3333%"
        >
          tab 1
        </div>
        <div
          class="zzc-tabs-ls-item"
          style="width:33.3333%"
        >
          tab 2
        </div>
        <div
          class="zzc-tabs-ls-item"
          style="width:33.3333%"
        >
          tab 3
        </div>
        <div
          class="zzc-tabs-ls-item"
          style="width:33.3333%"
        >
          tab 4
        </div>
        <div
          class="zzc-tabs-ls-item"
          style="width:33.3333%"
        >
          tab 5
        </div>
        <div
          class="zzc-tabs-ls-unline zzc-tabs-ls-unline-top zzc-tabs-ls-unline-am"
          style="width:33.3333%;transform:translate3d(0.0000%,0,0)"
        />
      </div>
    </div>
  </div>
  <div
    class="zzc-tabs-content-box zzc-tabs-content-box-horizontal"
  >
    <div
      class="zzc-tabs-content-wrap zzc-tabs-content-wrap-am"
      style="touch-action:pan-x pan-y;transform:translate3d(0, 0px, 1px);-webkit-transform:translate3d(0, 0px, 1px);-moz-transform:translate3d(0, 0px, 1px)"
    >
      <div
        class="zzc-tabs-content"
      >
        <div
          class="zzc-tabs-content-item"
        >
          <div
            class="item-content"
          >
            <p>
              this is 1 content
            </p>
          </div>
        </div>
        <div
          class="zzc-tabs-content-item"
        >
          <div
            class="item-content"
          >
            <p>
              this is 2 content
            </p>
          </div>
        </div>
        <div
          class="zzc-tabs-content-item"
        >
          <div
            class="item-content"
          >
            <p>
              this is 3 content
            </p>
          </div>
        </div>
        <div
          class="zzc-tabs-content-item"
        >
          <div
            class="item-content"
          >
            <p>
              this is 4 content
            </p>
          </div>
        </div>
        <div
          class="zzc-tabs-content-item"
        >
          <div
            class="item-content"
          >
            <p>
              this is 5 content
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
